packages/react-components/react-color-picker-preview/library/docs/Spec.md

```diff
@@ -2,62 +2,458 @@
+- `componentRef`
+- `showPreview`
+- `strings`
+- `tooltipProps`
+
+#### ColorArea component
+
+#### New props
+
+- `customStep`
+- `shape`
+- `step`
+- `valueX`
+- `valueY`
+
+#### Props supported
+
+- `color`
+
+#### ColorSlider component
+
+#### New props
+
+- `customStep`
+- `shape`
+- `orient`
+- `step`
+
+#### Props supported
+
+- `value`
+- `onChange`
 
-- _Migration from v8_
-- _Migration from v0_
+#### Props no longer supported with an equivalent functionality in ColorPicker V9:
+
+- `minValue` => Use `min` instead
+- `maxValue` => Use `max` instead
+- `type` => in case of support channels it'll be `channel` instead
+- `isAlpha` => use `AlphaSlider` component instead
+- `overlayColor` => part of `AlphaSlider` component
+- `thumbColor` => is calculated automatically or can be customized via CSS
+
+#### Property Mapping
+
+| v8 `ColorPicker` | v9 `ColorPicker`           |
+| ---------------- | -------------------------- |
+| `value`          | `value`                    |
+| `onChange`       | `onChange`                 |
+| `maxValue`       | `max`                      |
+| `minValue`       | `min`                      |
+| `shape `         |                            |
+| `type`           | `channel`                  |
+| `isAlpha`        | `AlphaSlider` component    |
+| `overlayColor`   | In `AlphaSlider` component |
+| `thumbColor`     |                            |
+|                  | `customStep`               |
+|                  | `shape`                    |
+|                  | `orient`                   |
+|                  | `step`                     |
 
 ## Behaviors
 
-_Explain how the component will behave in use, including:_
+### Mouse
+
+On `click` color is selected and color thumb is moved to the selected position. Color thumb should represent selected color.
+
+States:
+
+1. Rest
+2. Focus
+3. OnClick
+
+   Color is selected.
+
+### Keyboard Navigation
+
+In a `horizontal` orientation top/right arrows move to the right, bottom/left arrows move to the left.
+
+| Key                  | Result                                                         |
+| -------------------- | -------------------------------------------------------------- |
+| Arrows               | Color thumb is focused. Color is selected                      |
+| `Home/End/PgUp/PgDn` | Can be used another configuration for the step to move faster. |
+| `Tab`                | Navigation between color sliders and Color Area                |
+
+##### Color/Alpha Slider
+
+![Slider navigation](./assets/color-slider-navigation.jpg)
+
+##### ColorArea
 
-- _Component States_
-- _Interaction_
-  - _Keyboard_
-  - _Cursor_
-  - _Touch_
-  - _Screen readers_
+![ColorArea  navigation](./assets/color-area-navigation.jpg)
+
+### Screen readers
+
+On navigation and color select the screen reader should read the following:
+
+1. `aria-labelby` of the root component, e.g. "ColorPicker"
+2. `aria-roledescription` of the input slider, e.g. "2D slider"
```

Comment:
I'd suggest leaving this off -- since it's using range sliders under the hood, I think we can keep that approach and not override how the roles are exposed. If we decide to add more information about what's going on with the control, I'd prefer it to be in the label rather than in a `roledescription`